_data/errors/extraction/databases/mysql.yml

```diff
@@ -244,4 +246,19 @@ all:
 
        The version of the MySQL database is less than 5.6.2, which is the minimum version that supports binlog replication. Versions prior to 5.6.2 don't support the binlog replication feature, which is required to use Log-based Incremental Replication in Stitch.
     fix-it: |
-      The source MySQL database must be running MySQL version 5.6.2 or greater to use binlog replication. If you can upgrade your database version, you may be able to use binlog replication.
\ No newline at end of file
+      The source MySQL database must be running MySQL version 5.6.2 or greater to use binlog replication. If you can upgrade your database version, you may be able to use binlog replication.
+
+## Empty table error
+  - message: *empty-table-binlog
+    level: "critical"
+    category: "Log-based Incremental Replication"
+    category-doc: |
+      {{ link.replication.log-based-incremental | prepend: site.baseurl | append: "#log-based-incremental-replication-terminology" }}
```

Comment:
This doesn't link to the section that outlines the problem - ie: `#historical-replication-jobs`.